test/common/markdown/package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
As the tool isn't common and would only be used in this one test, I don't see any reason to move it to a common tools folder.